app.py

```diff
@@ -20,5 +21,28 @@ def Login():
             flash('Login Unsuccessful. Invalid Email/Password')
     return render_template('login.html',title='Login | SAC Portal, IIT Mandi',form=form)
 
+#Example  https://host/leaveclub?userID='B19188'&clubID='C10001'
+@app.route('/leaveclub', methods = ['POST'])
```

Comment:
Not sure about this comment. @vsvipul @Milind712000 what do you think about this?
Guard this function to check whether the user is logged in. If the user is logged in we don't need to store the userid.
If you are thinking about admin functionalities, we can have a different function for that.

Reply:
Ok, done, understood. 
I will do this.

Reply:
I will only take the clubID in the body, and take the userID from the session.